tests/test_format.py

```diff
@@ -78,6 +78,8 @@
     "parenthesized_context_managers",
 ]
 
+PREVIEW_CASES = []
```

Comment:
I guess `mypy` has some advanced analysis for the test case lists, since the hint isn't needed for other lists. Should I:

1) add hints for all case lists
2) add it just here and remove it when the first entry is added
3) add test infra later?

Reply:
I'd say add hints for all of them.